src/main/java/core/basesyntax/storage/StrategySubtraction.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.servise.FruitRecordDto;
+import core.basesyntax.servise.exception.InvalidValueOfAmountException;
+
+public class StrategySubtraction implements Strategy {
+    @Override
+    public int changeBalance(FruitRecordDto fruitRecordDto) {
+        Fruit newFruit = new Fruit(fruitRecordDto.getFruit());
+        Integer currentAmount = Storage.storageOfFruits.get(newFruit);
+        int newAmount = currentAmount - fruitRecordDto.getAmountOfFruit();
```

Comment:
What happens if method get() return - null? On 12 line you will have:  null + number

Reply:
I got this moment! Thanks a lot!))